pages/header/permutations.page.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+
+import { Button, Link } from '~components';
+import Header, { HeaderProps } from '~components/header';
+import createPermutations from '../utils/permutations';
+import PermutationsView from '../utils/permutations-view';
+import ScreenshotArea from '../utils/screenshot-area';
+
+const permutations = createPermutations<HeaderProps>([
+  {
+    variant: ['h1', 'h2', 'h3'],
+    info: [
+      <Link key="info" variant="info">
+        Info
+      </Link>,
+      null,
+    ],
+    description: ['description', null],
+    counter: ['counter', undefined],
```

Comment:
minor: I think it'd be nicer to put more realistic values in here, like `10` or `3/10`

Reply:
If I get a fast approval, I might do it ;)

Reply:
⚡️